packages/api/config.js

```diff
@@ -91,9 +94,11 @@ nconf.defaults({
   host: "localhost",
   webroot: "/var/www",
   upload_dir: "/tmp/spaship_uploads",
+  sse: "http://localhost:5000/sse/80",
```

Comment:
This should be broken down into an object with configurable attributes instead of a single hardcoded url.

Reply:
Fixed this comment.